src/containers/orders/cart/cart-item/cart-item.js

```diff
@@ -106,7 +118,7 @@ const CartItem = ({ item, setModalVisibility, setModalItem, cartOperations, prom
 
   const defaultProductImg = (
     <Link to={`${pathToProducts}/${cartItem?._id}`}>
-      <img className={styles.itemImg} src={`${IMG_URL}${itemFoto} `} alt='product-img' />
+      <img className={styles.itemImg} src={image} alt='product-img' />
```

Comment:
Isn't it better practice to use imageUrl naming for src attribute?

Reply:
fixed